homeassistant/components/lutron/__init__.py

```diff
@@ -53,7 +54,31 @@
 )
 
 
-def setup(hass: HomeAssistant, base_config: ConfigType) -> bool:
+async def async_setup(hass: HomeAssistant, base_config: ConfigType) -> bool:
+    """Set up the Lutron component."""
+    hass.data.setdefault(DOMAIN, {})
+
+    if DOMAIN in base_config:
+        lutron_configs = base_config[DOMAIN]
+        hass.async_create_task(
+            hass.config_entries.flow.async_init(
+                DOMAIN,
+                context={"source": config_entries.SOURCE_IMPORT},
+                # extract the config keys one-by-one just to be explicit
```

Comment:
Why do we need to be more explicit? The whole scheme is defined earlier in the file.